src/ui/Analytics/Analytics.ts

```diff
@@ -510,10 +513,6 @@ export class Analytics extends Component {
       this.accessToken = this.defaultEndpoint.accessToken;
```

Comment:
Since we are setting the `accessToken` on this line, what does that mean for the `subscribeToRenewal` call in the constructor, which is on the outdated instance?

Should we unsubscribe, and resubscribe every time we change the `this.accessToken` to avoid all subscriptions triggering?

Reply:
This is called in the constructor too, before the subscription happens. In the setupAccessToken() method.